pubspec.yaml

```diff
@@ -4,7 +4,7 @@ author: Dart Team <misc@dartlang.org>
 description: Obtain Access credentials for Google services using OAuth 2.0
 homepage: https://github.com/dart-lang/googleapis_auth
 environment:
-  sdk: '>=1.21.0 <2.0.0'
+  sdk: '>=2.0.0-dev.13.0 <2.0.0'
```

Comment:
Should probably bump the version of this package to at least 0.3.0, too, in case we need to release an update that supports stable.

Reply:
I'd like to avoid it, honestly. The public API hasn't changed at all.

We won't need to change consuming packages to let folks on dev-16+ get this update – which is ideal.